pkg/reconciler/pipelinerun/resources/validate_params_test.go

```diff
@@ -262,13 +268,18 @@ func TestValidateObjectParamRequiredKeys_Valid(t *testing.T) {
 					"key1": {Type: "string"},
 					"key2": {Type: "string"},
 				},
+				Default: &v1beta1.ParamValue{
```

Comment:
I think we'd want to keep the existing test case and add a new one where one of the keys is provided by the default

Reply:
SG! Reverted the old test back and add this new one.